### ISSUE TITLE
Bounding Box RMShape

### ISSUE THREAD

**Issue by onato** [mapbox/DEPRECATED-mapbox-ios-sdk#170]
My RMShape is disappearing when I pan the map. I am using 0.5.0. 

```
NSLog(@"Bounding: %f,%f,%f,%f",path.pathBoundingBox.origin.x, path.pathBoundingBox.origin.y, path.pathBoundingBox.size.width, path.pathBoundingBox.size.height);
//Bounding: 0.000000,0.000000,0.000000,0.000000
```

I guess this has something to do with my GCD stuff...

```
    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0), ^{
        [path performBatchOperations:^(RMShape *aPath){
            for (TSLocation *location in locations)
            {
                dispatch_async(dispatch_get_main_queue(), ^{
                    [aPath addLineToCoordinate:location.coordinate];
                });
            }
            int64_t delayInSeconds = 8.0;
            dispatch_time_t popTime = dispatch_time(DISPATCH_TIME_NOW, delayInSeconds * NSEC_PER_SEC);
            dispatch_after(popTime, dispatch_get_main_queue(), ^(void){
                NSLog(@"Bounding: %f,%f,%f,%f",path.pathBoundingBox.origin.x, path.pathBoundingBox.origin.y, path.pathBoundingBox.size.width, path.pathBoundingBox.size.height);
            });
        }];
    });
```

**Comment by incanus** [comment]
Your GCD code looks ok. Give the develop branch a try, as there was a fix post-0.5.1 which addressed shape bounds issues that might also solve what you are seeing.

**Comment by incanus** [comment]
Gonna close for now; feel free to reopen if this exists with the latest code.
